tps-core/src/main/scala/tps/ArgHandling.scala

```diff
@@ -63,7 +63,7 @@ object ArgHandling {
       opt[String]("solver") action { (v, o) =>
         o.copy(synthesisOptions = 
           o.synthesisOptions.copy(solver = v)) } text(
-          "solver type (naive, bilateral or dataflow)")
+          "solver type (only 'dataflow' is currently supported)")
```

Comment:
I'm thinking that maybe I should altogether remove those two from command-line arguments (and update documentation). How does that sound?

Reply:
If we remove these two arguments and they were specified at the command line, would TPS exit with an error?  Or would they be ignored?

I was thinking about deprecating them for a version or two such that the command line arguments can be parsed but do not have an effect.  I'm imagining a situation in which someone has a script that calls TPS and provides these arguments as part of the script.  The new exception message `Only the dataflow solver is supported.` should be enough for the user to notice that they need to update their solver.  A potential corner case could be if someone updates the solver but leaves the slack or bitvect arguments and doesn't realize those need to be removed.

Reply:
If we remove these arguments from the command-line, providing them would lead to an error in argument parsing at start-up, so I think this would be a safe change. Meanwhile, as you indicated, it might be best to also indicate which changes to make (maybe in the README) in case a user was providing these arguments—although I think this is quite unlikely to have happened :)

Reply:
That sounds good to me.

@ajshedivy is working with me on a wrapper script that calls TPS.  We're aware of the changes being made here, so it won't cause any problems for us.

Reply:
Thanks, Tony. I removed these two arguments from the CLI.